src/lib/shopping-fpti/event-handlers.js

```diff
@@ -23,9 +24,14 @@ export function eventSinfoBuilderInit(config : Config) : Object {
   function constructSinfoPayload(payload : Object) : ?string {
     const shopperConfig = config.shoppingAttributes || {};
 
+    const shouldCaptureData = window.__pp__shopping__ && window.__pp__shopping__.capturePageData;
+
+    const capturedData = shouldCaptureData ? capturePageData() : {};
+
     const enrichedPayload = filterAttributesForSinfoPayload({
       ...payload,
-      ...shopperConfig
+      ...shopperConfig,
+      ...capturedData
```

Comment:
`capturedData` will it have specific root attribute name where the scraped JSON data will be added. Like, e.g
`ogTags`: {}

Reply:
Can you share a SDK event with OG tags embedded